KIPs/kip-fungible_token.md

```diff
@@ -0,0 +1,242 @@
+This standard provides basic functionality to transfer tokens.
+
+## Motivation
+<!--The motivation is critical for KIPs that want to change the Klaytn protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the KIP solves. KIP submissions without sufficient motivation may be rejected outright.-->
+A standard interface allows any token on Klaytn to be re-used by other applications: from wallets to decentralized exchanges.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Klaytn platforms (klaytn). -->
+This document derived heavily from Ethereum's [ERC-20 token standard](https://eips.ethereum.org/EIPS/eip-20) written by Fabian Vogelsteller and Vitalik Buterin.
+Comparing with ERC-20 token standard, this token standard has two differences:
+- includes three more optional functions (mint, burn, and burnFrom)
+- MUST trigger transfer events when minting and burning tokens
+
+### Summary of Methods and Events
+The table below is a summary of methods.
+The prototype uses the syntax from Solidity `0.4.24` (or above).
+If the optional field is not marked, the function must be implemented.
+
+|Name|Optional|Prototype|
+|---|:---:|---|
+|[totalSupply](#totalsupply)| |function totalSupply() public view returns (uint256)|
+|[balanceOf](#balanceof)| |function balanceOf(address _owner) public view returns (uint256 balance)|
+|[transfer](#transfer)| |function transfer(address _to, uint256 _value) public returns (bool success)|
+|[transferFrom](#transferfrom)| |function transferFrom(address _from, address _to, uint256 _value) public returns (bool success)|
+|[approve](#approve)| |function approve(address _spender, uint256 _value) public returns (bool success)|
+|[allowance](#allowance)| |function allowance(address _owner, address _spender) public view returns (uint256 remaining)|
+|[name](#name)|O|function name() public view returns (string)|
+|[symbol](#symbol)|O|function symbol() public view returns (string)|
+|[decimals](#decimals)|O|function decimals() public view returns (uint8)|
+|[mint](#mint)| O |function mint(address _to, uint256 _value) public returns (bool) |
+|[burn](#burn)| O |function burn(uint256 __value) public |
+|[burnFrom](#burnfrom)| O |function burnFrom(address _from, uint256 _value) public |
+
+The table below is a summary of events.
+The prototype uses the syntax from Solidity `0.4.24` (or above).
+All the following events must be implemented.
+
+|Name|Prototype|
+|---|---|
+|[Transfer](#transfer-1)|event Transfer(address indexed _from, address indexed _to, uint256 _value)|
+|[Approval](#approval)|event Approval(address indexed _owner, address indexed _spender, uint256 _value)|
+
+
+### Methods
+
+#### name
```

Comment:
We also need to reorder these descriptions. :)

Reply:
I got it. :)

Reply:
I moved and added descriptions  about optional function.